reflex/vars.py

```diff
@@ -1958,6 +1963,43 @@ def _determine_var_type(self) -> Type:
         return Any
 
 
+def computed_var(
+    fget: Callable[[BaseState], Any] | None = None,
+    fset: Callable[[BaseState, Any], None] | None = None,
+    fdel: Callable[[BaseState], Any] | None = None,
+    doc: str | None = None,
+    initial_value: Any | None = None,
+    **kwargs,
```

Comment:
lets explicitly define `cache: bool = False` so users can create a cached var like `@rx.var(cached=True)`

then update the `cached_var` down below to essentially be a `functools.partial` of this with `cached=True` already set?

that would allow users to also write `@rx.cached_var(initial_value="foo")` and keep both decorators consistent